src/zope/i18n/locales/xmlfactory.py

```diff
@@ -22,9 +22,9 @@
 from zope.i18n.locales import LocaleOrientation, LocaleDayContext
 from zope.i18n.locales import LocaleMonthContext, calendarAliases
 from zope.i18n.locales.inheritance import InheritingDictionary
-from .._compat import _u
 
-_BLANK = _u('')
+
+_BLANK = u''
```

Comment:
I would suggest dropping _BLANK.  It was introduced merely because we couldn't use `u''` on Python 3.early.